pkg/systemd/page-status.jsx

```diff
@@ -47,20 +49,32 @@ export class PageStatusNotifications extends React.Component {
     }
 
     render() {
-        // Just this one for now
-        const page = "system/services";
-        const status = page_status.get(page);
-        if (status && status.type && status.title) {
-            this.props.toggle_label && this.props.toggle_label(true);
-            const jump = () => cockpit.jump("/" + page);
-            return (
-                <>
-                    <span className={icon_class_for_type(status.type)} />
-                    <a role="button" tabIndex="0" onClick={jump}>{status.title}</a>
-                </>);
-        } else {
-            this.props.toggle_label && this.props.toggle_label(false);
-            return null;
-        }
+        // Explicit whitelist for now, until we can get a dynamic list
+        return ["system/services", "updates"].map(page => {
+            const status = page_status.get(page);
+            if (status && (status.type || status.details) && status.title) {
+                this.props.toggle_label && this.props.toggle_label(true);
```

Comment:
`toggle_label` is not used and doesn't make sense anymore, I'd say.  Let's remove it.

Reply:
Ack, done in a separate commit.